### PR TITLE
clean log by adding debug mode which start with npm run debug

### PR DESCRIPTION
**Pull request by henryzbw**
I don't think there is any point to add error level since I think the error should always be shown. 
If you want to show the logs, run `npm run debug`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "babel-node src/index.js",
+    "debug": "babel-node src/index.js 20",
```
Comment (frankobe) [inline package.json]: `loglevel` can be config in `.ENV`

File: src/index.js [hunk 2]
```diff
@@ -32,9 +41,11 @@ server.use(restify.plugins.bodyParser({ mapParams: true }));
 server.use(restify.plugins.queryParser());
 server.on('after', (req, res, route, err) => {
   if (route) {
-    console.log(`${route.methods[0]} ${route.spec.path} ${res.statusCode}`);
+    if(IS_DEBUG_MODE){
```
Comment (frankobe) [inline src/index.js]: change `console.log` to `console.info` is the correct route to go

File: package.json (hunk 1 above)
Comment (henryzbw) [reply package.json]: .env file is excluded for git commit,  if we want to use it, will everyone need to config it on their own?

**Comment by frankobe** [comment]
@henryzbw refer to https://blog.risingstack.com/node-js-logging-tutorial/ on how to setup logger properly

**Pull request closed without merging**